beacon_node/network/src/router/mod.rs

```diff
@@ -68,7 +68,8 @@ impl<T: BeaconChainTypes> Router<T> {
 
         // Initialise a message instance, which itself spawns the syncing thread.
         let processor = Processor::new(
-            runtime_handle,
+            // TODO: spawn_blocking here
```

Comment:
why the TODO?

Reply:
Wanted to do the `spawn_blocking`  stuff in another PR. Just so I don't forget.